### ISSUE TITLE
Archive deployments

### ISSUE THREAD

**Issue by vwkd** [denoland/deploy_feedback#311]
It would be great if we could archive deployments similar to archiving GitHub repositories.

Sometimes one wants to keep a deployment without intending to change it in the future, so deletion is not an option. Currently, those deployments fill up the "Projects" page. Also they risk being accidentally broken because Deno Deploy doesn't block any changes (e.g. commit push).

**Comment by lucacasonato** [comment]
Would you want archived projects to still serve traffic?

**Comment by vwkd** [comment]
Yes, it should still serve traffic. Otherwise there's not much reason to not delete it altogether.
